models/game_play.py

```diff
@@ -12,12 +12,14 @@ def __init__(self):
                 Ship("BATTLESHIP", 4), 
                 Ship("CRUISER", 3), 
                 Ship("SUBMARINE", 3), 
+                Ship("FRIGATE", 3),
                 Ship("DESTROYER", 2)
             ],
             'cpu': [ 
                 Ship("BATTLESHIP", 4), 
                 Ship("CRUISER", 3), 
                 Ship("SUBMARINE", 3), 
+                Ship("FRIGATE", 3),
                 Ship("DESTROYER", 2)
             ]
         }
```

Comment:
oooh ya know - we could probably create a loop here that runs the same init Ship lines for all ships for the player and the cpu. Let's see if a for loop causes any issues.